storage/url/url.go

```diff
@@ -347,3 +348,12 @@ func parseNonBatch(prefix string, key string) string {
 func hasGlobCharacter(s string) bool {
 	return strings.ContainsAny(s, globCharacters)
 }
+
+func (u *URL) GetCopySource() string {
```

Comment:
Since `storage.URL` is a generic abstraction for all storage operations, I recommend using a more generic method name such as `EscapedPath()`, etc. 

```suggestion
func (u *URL) EscapedPath() string {
```

Reply:
I updated as your suggestion